test/ex_wrapplog_test.exs

```diff
@@ -6,46 +6,55 @@ defmodule ExWrapplogTest do
   test "log method" do
     Application.put_env(:ex_wrapplog, :level, :debug)
     assert capture_io(fn  -> ExWrapplog.log(:error, "This is an error") end)
-      == "ERROR {\"msg\":\"This is an error\",\"level\":\"error\"}\n"
+      == "{\"timestamp\":\"2017-03-28T13:46:06.989172Z\",\"service\":\"\",\"msg\":\"This is an error\",\"level\":\"error\"}\n"
```

Comment:
(comment related to all tests) how can you know the order? I avoided similar test-cases because I assumed maps/dicts using some kind of hashing, therefore giving a somewhat random order. So for me only way would be to use a sorted-map or so. Do you know it works? (In next json-version to?:P )